stable/external-dns/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.19.1
+version: 2.19.2
```

As this is a new feature, can you bump the minor version?
```suggestion
version: 2.20.0
```